src/Freya.Core/Integration.fs

```diff
@@ -43,29 +39,26 @@ type OwinAppFunc =
 [<RequireQualifiedAccess>]
 [<CompilationRepresentation (CompilationRepresentationFlags.ModuleSuffix)>]
 module OwinAppFunc =
-    open Freya.Core.Functions
 
-    /// Converts a <see cref="Freya{T}" /> computation to an <see cref="OwinAppFunc" />.
-    [<CompiledName ("FromFreya")>]
-    let fromFreya (freya: Freya<_>) =
+    /// Converts a <see cref="Core{T}" /> computation to an <see cref="OwinAppFunc" />.
+    [<CompiledName ("FromCore")>]
+    let fromCore (core: Core<_>) =
```

Comment:
Yes, I do agree about having Freya in the name here too.

So, in terms of type names and computation expression names, I'm thinking of options as:
{ freya, freyaMachine, freyaRouter } (what it was originally! But i'd like to still strip out some of the Freya from type names... Maybe that just doesn't really work. Maybe I'll settle for all of the publicly facing types having Freya prefixes...
{ owin, machine, router } not too sure on that at all. It would make the Core library type Owin<'a> and I'm not sure about that. A bit too OWIN-y perhaps.

Hmmm. Not great, I think I'll go back for now on these, but have a play with tidying up the type names further.